index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css" />
+  <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+  />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+  <title>Cedrich's Portfolio</title>
+</head>
+<body>
+
+  <header>
+    <nav class="navbar">
+      <a href="#" class="navItem"><h2>Welcome</h2></a>
+      <a href="#" class="navItem"><i class="fa fa-bars"></i></a>
+    </nav>
+  </header>
+  
+  <main>
+    <section class="content-container">
+      <h1>
+        Hey there, I'm Cedrich<br /><span class="profession"
+          >I'm a software developer.</span
+        >
+      </h1>
+      <p>
+        I can help you build a product , feature or website. Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hestiate to contact me.
+      </p>
+      <ul class="socilaLinks">
+        <li>
+          <a href="#" target="_blank"><img src="/images/twitter.png" alt="twitter" /></a>
```

Comment:
- [ ] Kindly fix your icons
<img width="986" alt="Screenshot 2022-12-03 at 16 18 10" src="https://user-images.githubusercontent.com/81444338/205467776-39b306f8-c3b1-4d47-8e63-8688d772386c.png">

Reply:
I notice that we have to clone the project to see the icons, so they are working fine :+1: , Good job @ceddddy